docs/src/main/sphinx/release/release-365.md

```diff
@@ -0,0 +1,147 @@
+* Improve performance of decimal ``avg`` aggregation. ({issue}`9738`)
+* Improve query performance over Parquet data with timestamp encoded using `int64` representation in Parquet. ({issue}`9414`)
+* Improve dynamic partition pruning efficiency. ({issue}`9866`, {issue}`9869`)
+* Improve query performance on partitioned tables or tables with small files. ({issue}`9979`)
+* Fix reporting of number of read bytes for tables using ``ORC`` file format. ({issue}`10048`)
+* Account memory used for tracking information about deleted rows when reading from Hive ACID tables. ({issue}`9914`, {issue}`10070`)
+* Fix `REVOKE GRANT OPTION` to revoke only the grant option instead of revoking entire privilege. ({issue}`10094`)
+* Fix bug that skips writing bloom filters for ORC files. ({issue}`9792`)
+* Fix bug where incorrect rows were deleted when deleting from a transactional table that has original files (before the first major compaction). ({issue}`10095`)
+* Fix delete and update failure when changing a table after a major compaction. ({issue}`10120`)
+* Fix incorrect results when decoding decimal values in Parquet reader. ({issue}`9971`)
+* Fix ``hive.dynamic-filtering.wait-timeout`` not having any effect. ({issue}`10106`)
+* Fix failure when reading Parquet data if column indexes are enabled. ({issue}`9890`, {issue}`10076`)
+
+## Iceberg connector
+
+* Add support for storing and reading UUID nested in `row`, `array` or `map` type. ({issue}`9918`)
+* Use Iceberg's `schema.name-mapping.default` table property for scanning files
+  with missing Iceberg ids. This aligns Trino behavior on migrated files with the
+  Iceberg spec. ({issue}`9959`)
+* Use ZSTD compression by default. ({issue}`10058`)
+* Add read-only security option which can be enabled by setting the configuration `iceberg.security=read-only`. ({issue}`9974`)
+* Change schema of `$partitions` system table to avoid conflicts when table name contains a column named `row_count`, `file_count`
+  or `total_size`, or when a column is used for partitioning for part of table data, and it not used for partitioning in some other
+  part of the table data. ({issue}`9519`, {issue}`8729`).
+* Improve query performance over Parquet data with timestamp encoded using `int64` representation in Parquet. ({issue}`9414`)
+* Improve query performance for certain queries with complex predicates. ({issue}`9309`)
+* Reduce resource consumption and create bigger files when writing to an Iceberg table with partitioning. Bigger files are more efficient to query later. ({issue}`9826`)
+* Improve performance for queries on nested data through dereference pushdown. ({issue}`8129`)
+* Write correct `file_size_in_bytes` in manifest when creating new ORC files. ({issue}`9810`)
+* Fix query failures that could appear when reading Parquet tables which contained ROW columns that were subject to schema evolution. ({issue}`9264`)
+* Fix failure caused by stale metadata in the `rollback_to_snapshot` procedure. ({issue}`9221`)
```

Comment:
This is the wrong issue. I don't know the right one.

Reply:
It's https://github.com/trinodb/trino/pull/9921